app/views/workflow_executions/_files.html.erb

```diff
@@ -1,60 +1,35 @@
-<div class="relative overflow-x-auto" data-turbo-temporary>
+<div class="relative overflow-x-auto">
   <table
     id="files-table"
-    class="
-      w-full
-      text-sm
-      text-left
-      rtl:text-right
-      text-slate-500
-      dark:text-slate-400
-      whitespace-nowrap
-    "
+    class="w-full text-sm text-left table-auto  rtl:text-right text-slate-500 dark:text-slate-400 whitespace-nowrap"
   >
     <thead
-      class="
-        text-xs
-        text-slate-700
-        uppercase
-        bg-slate-50
-        dark:bg-slate-700
-        dark:text-slate-400
-      "
+      class="text-xs uppercase  text-slate-700 bg-slate-50 dark:bg-slate-700 dark:text-slate-400"
     >
       <tr>
         <th
           scope="col"
-          class="
-            px-6
-            py-3
-            text-left
-            bg-slate-50
-            dark:bg-slate-700
-            sticky
-            left-0
-            z-10
-          "
+          class="sticky left-0 z-10 px-6 py-3 text-left  bg-slate-50 dark:bg-slate-700"
```

Comment:
Extra space in the classes

Reply:
Appears to be added by the TailwindCSS formatter